Visualization/Adaptive Retrieval/PLOTS - Sheet15.csv

```diff
@@ -0,0 +1,7 @@
+FLOPS,MAP@10

```

Comment:
@adityakusupati do we need to provide all these CSV files and scripts here?

Reply:
I don't mind but if we want to we need to clean up the names, PLOTS - Sheet blah doesn't work. It is generally not needed, but if someone likes our plots, they can reuse and add to our plots.